.github/workflows/mirror.yml

```diff
@@ -11,4 +11,4 @@ jobs:
     steps:
       - uses: zofrex/mirror-branch@v1
         with:
-          target-branch: main
+          target-branch: master
```

Comment:
Are you sure we shouldn't create a third, temporary branch and mirror main to this branch just to make sure it all works and doesn't mess up the master branch somehow, because we're not experienced with changing the default branch of a large repo like this?